src/app/persistence/PascalString.h

```diff
@@ -89,9 +93,13 @@ class PascalBuffer
         static_assert(N <= kInvalidLength);
     }
 
+    /// Allocates a pascal buffer of the given size.
```

Comment:
Nothing his being allocated here... should this say something like "Interprets the given span as a pascal buffer" ?